cli/validation/index.md

```diff
@@ -194,6 +231,19 @@ batch summary on the test machine is shown below for reference:
 meaning the software took one minute and forty seconds to process one thousand
 and five hundred files.
 
+<a name="archive validation"></a>Archive validation
+------------------------------------------------
+
+There is also possible validate multiple archived PDF documents. If input file is zip archive, verapdf recursively validate PDF files in all zip folders.
```

Comment:
It is also possible to validate multiple PDF documents within a ZIP archive. If an input file has ZIP format, veraPDF recursively scans and validates PDF files in all subfolders within the archive. For example